dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -36,26 +36,32 @@
         "opitem": "Github Access Token Service Bot",
         "opfield": ".password",
     },
+    "ZENHUB_ACCESS_TOKEN": {
+        "opitem": "Zenhub Access Token",
+        "opfield": ".password",
+    },
 }
 
 with DAG(
     dag_id=f"atd_service_bot_issues_to_dts_portal_{DEPLOYMENT_ENVIRONMENT}",
     default_args=default_args,
-    schedule_interval="0 5 * * *",
+    schedule_interval="0 5 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
```

Comment:
I like the pattern where DAGs are unscheduled by default locally, so I am bringing that in here.

Reply:
i like it too, thanks for bringing it in